### PR TITLE
Fixed manager and node - api and websockets, added debugger endpoints 

### PR DESCRIPTION
**Pull request by kkingstoun**

### CODE REVIEW

File: .gitignore
Comment (MathieuMoalic) [inline .gitignore:1]: schema.yml should be in the repo, remove from gitignore

File: manager/amuman/routing.py
Comment (MathieuMoalic) [inline manager/amuman/routing.py:1]: remove the file if it's empty

File: manager/manager/forms.py
Comment (MathieuMoalic) [inline manager/manager/forms.py:1]: Remove if unused

File: manager/manager/routing.py
Comment (MathieuMoalic) [inline manager/manager/routing.py:1]: delete unused files

File: manager/manager/serializers.py
Comment (MathieuMoalic) [inline manager/manager/serializers.py:1]: if chatgpt can write the comments in english, it might help me understand

Comment (MathieuMoalic) [reply manager/manager/serializers.py:1]: also there are a lot of dupa in there

File: manager/manager/views.py
Comment (MathieuMoalic) [inline manager/manager/views.py:1]: A lot of commented out code in there

File: node/amuman_node/main.py [hunk 1]
```diff
@@ -196,3 +198,10 @@ def entrypoint() -> None:
         asyncio.run(NodeClient().start())
     except KeyboardInterrupt:
         log.warning("Caught KeyboardInterrupt (Ctrl+C). Shutting down...")
+
+
+if __name__ == "__main__":
```
Comment (MathieuMoalic) [inline node/amuman_node/main.py:203]: I assume this is because of debugpy, can't you just call the entrypoint function here?

**Review by MathieuMoalic — CHANGES_REQUESTED** [review]
just a few tiny changes and you can merge. The logic seems good

File: .gitignore
Comment (kkingstoun) [reply .gitignore:1]: OK.

**Review by MathieuMoalic — APPROVED** [review]

**Pull request merged**